instance/templates/instance/ansible/vars.yml

```diff
@@ -26,19 +31,11 @@ CMS_HOSTNAME: '{{ instance.studio_domain_nginx_regex }}'
 EDXAPP_SESSION_COOKIE_DOMAIN: ''
 
 # Nginx
-{% if instance.protocol == 'https' %}
-NGINX_ENABLE_SSL: true
-NGINX_REDIRECT_TO_HTTPS: true
-{% else %}
+NGINX_SET_X_FORWARDED_HEADERS: false
+
+# SSL is handled on the load balancer, and the appservers are HTTP only.
 NGINX_ENABLE_SSL: false
```

Comment:
Is there any reason not to set this to true so we can optionally connect to the VM using HTTPS and test it? (Which would use the snakeoil cert, but an SSL warning is fine for internal testing). Use case is to debug issues or verify fixes that could be related to HTTPS (e.g. cross-domain asset loading) without activating the VM.